sdk/attestation/azure-security-attestation/azure/security/attestation/aio/_administration_client_async.py

```diff
@@ -4,320 +4,662 @@
-        self._client = AzureAttestationRestClient(credential, instance_url, **kwargs)
+        self._config = AttestationClientConfiguration(**kwargs)
+        self._client = AzureAttestationRestClient(credential, endpoint, **kwargs)
         self._statelock = Lock()
         self._signing_certificates = None
 
+        self._signing_key = None
+        self._signing_certificate = None
+
+        signing_key = kwargs.pop("signing_key", None)
+        signing_certificate = kwargs.pop("signing_certificate", None)
+        if signing_key or signing_certificate:
+            self._signing_key, self._signing_certificate = validate_signing_keys(
+                signing_key, signing_certificate
+            )
+
     @distributed_trace_async
     async def get_policy(
-        self,
-        attestation_type, #type: AttestationType
-        **kwargs: Any
-        ) -> AttestationResponse[str]:
-        """ Retrieves the attestation policy for a specified attestation type.
+        self, attestation_type: AttestationType, **kwargs: Any
+    ) -> AttestationGetPolicyResult:
+        """Retrieves the attestation policy for a specified attestation type.
 
-        :param azure.security.attestation.AttestationType attestation_type: :class:`azure.security.attestation.AttestationType` for 
+        :param azure.security.attestation.AttestationType attestation_type: :class:`azure.security.attestation.AttestationType` for
             which to retrieve the policy.
-        :return: Attestation service response encapsulating a string attestation policy.
-        :rtype: azure.security.attestation.AttestationResponse[str]
+        :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
+        :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+            if the token is invalid, the `validation_callback` function should throw
+            an exception.
+        :paramtype validation_callback: ~typing.Callable[[AttestationToken, AttestationSigner], None]
+        :keyword bool validate_signature: if True, validate the signature of the token being validated.
+        :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+        :keyword str issuer: Expected issuer, used if validate_issuer is true.
+        :keyword float validation_slack: Slack time for validation - tolerance applied
+            to help account for clock drift between the issuer and the current machine.
+        :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+        :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
+
+        :return: A tuple containing the attestation policy and the token returned
+            by the service..
+
+        :rtype: AttestationGetPolicyResult
```

Comment:
Don't use the full type in the docstrings, this is what's generated and put on msdocs so we want the types to be easily found, and the aliases are not.

Reply:
I switched to use the full type in the docstrings (with a ~ before it) because the API doc generator didn't seem to generate links on partial type names. 

And I tried to switch to the type alias because the Tuple problem wasn't fixed with the full type name.